src/track.c

```diff
@@ -453,4 +469,51 @@ void tracking_send_state()
 
 }
 
+/** Parse a string describing the tracking loop filter parameters into
+    the loop_params_stage structs. */
+static bool parse_loop_params(struct setting *s, const char *val)
+{
+  /** The string contains loop parameters for either one or two
+      stages.  If the second is omitted, we'll use the same parameters
+      as the first stage.*/
+
+  if (s->type->from_string(s->type->priv, s->addr, s->len, val)) {
```

Comment:
Should try to parse `val` first.  If the setting is rejected, `loop_params_string` will still be written over, and may be saved to the settings file if requested by the user.